### ISSUE TITLE
Multiline editing

### ISSUE THREAD

**Issue by ardagnir** [ardagnir/athame#50]
Right now, multiline commands are effectively broken into separate lines by Athame. This is baked pretty deeply into how Athame works, but it would be nice to change because Vim would be especially useful for editing multiline commands.

**Comment by henrebotha** [comment]
Oof, yeah, this is just about a dealbreaker for me at the moment. I need to do work with big curl commands, which are just far more palatable in multiline format.

Can you state in a sentence or two what would need to change to support this @ardagnir?

**Comment by ardagnir** [comment]
I’ve actually been working on this a little bit since the last release. It mostly involves trying to get some way to have vim track which lines go together, and I currently have something partially, but not really, working using folds that I might push out in an experimental branch soon if I can get it working better.

**Comment by wahrwolf** [comment]
how about we use folds for that?
vim is pretty good in keeping track of them...

**Comment by resolritter** [comment]
@ardagnir do you still have the experimental branch? I'm interested in seeing how it went, even if it's unfinished

**Comment by ardagnir** [comment]
No, I was dumb and never uploaded it. :( Now the only copy is in storage >2000 miles from me and I won't be able to get it any time soon because of covid
